screens/Discover.js

```diff
@@ -57,7 +57,10 @@ class Discover extends Component {
         onCompletion(references)
       });
     } else {
-      onCompletion(null)
+      if (this.state.refreshing) {
```

Comment:
Here this is not an internet connection error, this means that the firebase was not able to retrieve user info, which may be related to firebase issues.